server/run.py

```diff
@@ -42,13 +42,21 @@ def bundle_js():
             return redirect('//localhost:3000/bundle.js')
         else:
             return app.send_static_file('bundle.js'), 200
-
+    
     @app.before_first_request
     def create_tables():
         db.create_all()#must be run before interacting with database
 
-    return app
+    # add CORS 
+    @app.after_request
+    def after_request(response):
+        response.headers.add('Access-Control-Allow-Origin', 'http://localhost:3000')
```

Comment:
This is a security flaw as it is currently is implemented. Anyone will be able to start a local server at port 3000 and make false requests. This should be prefaced by checking if the server is in debug mode via `current_app.debug` boolean.

See this example:
https://github.com/giancarlopernudisegura/cmput404/blob/master/server/run.py#L41-L42

Reply:
I am expecting to change this to an environment variable in another PR. like an environment that will be PREACT_URL or something along those lines. Like at the beginning of the run.py, we will have 
```python
current_url = PREACT_DEBUG_URL if current_app.debug else PREACT_PROD_URL
```
where PREACT_DEBUG_URL is localhost:3000, and PREACT_PROD_URL is our url.

It seems redundant to have to check `current_app.debug` boolean all the time request like this happen. Let me know your thoughts